unit_testing/cli/test_cli_sct_compute_hausdorff_distance.py

```diff
@@ -1,14 +1,22 @@
-from pytest_console_scripts import script_runner
 import pytest
 import logging
 
+from spinalcordtoolbox.scripts import sct_compute_hausdorff_distance
+
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.script_launch_mode('subprocess')
-def test_sct_compute_hausdorff_distance_backwards_compat(script_runner):
-    ret = script_runner.run('sct_testing', '--function', 'sct_compute_hausdorff_distance')
-    logger.debug(f"{ret.stdout}")
-    logger.debug(f"{ret.stderr}")
-    assert ret.success
-    assert ret.stderr == ''
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+def test_sct_compute_hausdorff_distance_values_against_threshold():
+    """Run the CLI script and verify computed distances are all within a given threshold.
+    TODO: Distances are all 0.0. Is this test checking anything useful?"""
+    sct_compute_hausdorff_distance.main(argv=['-i', 't2s/t2s_gmseg_manual.nii.gz', '-d', 't2s/t2s_gmseg_manual.nii.gz'])
+
+    with open('hausdorff_distance.txt', 'r') as f:
+        hausdorff_distance_lst = []
+        for line in f.readlines():
+            if line.startswith('Slice'):
+                hausdorff_distance_lst.append(float(line.split(': ')[1].split(' -')[0]))
+
+    assert max(hausdorff_distance_lst) <= 1.0
```

Comment:
I think this is because the original test was checking the distance between one file and itsel:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/998226f1e654961da6bdf77ad083b5f74013beb2/testing/test_sct_compute_hausdorff_distance.py#L18

So, it makes sense that the computed distances are 0. If we stick to comparing one file with itself, then the assertion should probably be `assert np.all(hausdorff_distance_list == 0)`. 

(Otherwise, we could find two different files to compare against one another, but... that would be a more involved test, I think.)

Reply:
In that case, rename the test `test_sct_compute_null_hausdorff_distance` for now and we can find a fuller test later. Testing the null cases is important too, but it shouldn't be the only thing we do!

Reply:
Done in 03ea808.